backend/src/chats/chat-user-permissions/entities/chat-user-permission.entity.ts

```diff
@@ -10,10 +11,27 @@ import {
 } from 'typeorm';
 import { User } from '../../../users/user/entities/user.entity';
 import { Chat } from '../../chat/entities/chat.entity';
-import { Permission } from '../../permissions/entities/permission.entity';
+
+export enum permissionsEnum {
+	READ = 'read',
+	POST = 'post',
+	LEFT = 'left',
+	BLOCKED = 'blocked',
+	MANAGE_USERS = 'manage_users',
+	EDIT_SETTINGS = 'edit_settings',
```

Comment:
what do the manage_users mean?
and is the edit settings the default setting?

Reply:
"manage users" is that you can access the page mentioned in this task: #138 

"edit settings" is that you can access the page mentioned in this task: #135